yarn.lock

```diff
@@ -10291,7 +10300,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"moment@npm:^2.19.3, moment@npm:^2.29.4":
+"moment@npm:^2.19.3":
```

Comment:
Dang, looks like this is still transitively required through the logging library that [`passport-azure-ad`](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/maintenance/passport-azure-ad) uses.